content/docs/examples/httpendpoints.md

```diff
@@ -11,36 +11,98 @@
 # Build an HTTP API Endpoint
 {{<header_divider>}}
 
-HTTP endpoints have become one of the most ubiquitous ways of delivering software to end users. Cdev provides a simple interface for creating HTTP APIs and connecting them to `Serverless Functions`. 
+`HTTP` endpoints have become one of the most ubiquitous ways of delivering software to end users. Cdev provides a simple interface for creating `HTTP APIs` and connecting them to `Serverless Functions`. You can find more information about `HTTP` in the **[official documentation](https://httpwg.org/)**.
+
+{{<tool_tip key="info" summary="Set Up">}}
+It is assumed that you have already installed the Cdev SDK, which has a pre-filled src/hello_world/resources.py file. This example starts with an empty src/hello_world/resources.py file. You can empty the contents of your file or replace the neccessary lines of code as you go along.
+{{</tool_tip>}}
 
 {{<break 1>}}
 ## Create an API
-{{<codesnippet `/source_code/http_examples/simple_api_example.py`>}}
+
+Open the `src/hello_world/resources.py` file and start by importing the following resources:
+{{<codesnippet `/source_code/http_examples/simple_api_example_resources.py`>}}
 
 
 {{<break 1>}}
-## Create routes
-{{<codesnippet `/source_code/http_examples/simple_api_example_get.py`>}}
+## Create a route
+Next, add lines `lines 8, 10 and 12` to define your project, `API` and create your route.
```

Comment:
Need to remove one of the "lines" in line 29

Reply:
Good catch!